### PR TITLE
Tidied HTML indents

### PR DESCRIPTION
**Pull request by josh-jacobsen**
Hi team, 

Can you please check out the HTML indenting and let me know if it's good to go.

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+    <section class="intro" id="zen-intro">
+
+      <header role="banner">
+        <h1>CSS Zen Garden</h1>
+        <h2>The Beauty of <abbr title="Cascading Style Sheets">CSS</abbr> Design</h2>
+      </header>
+
+      <div class="summary" id="zen-summary" role="article">
+        <p>A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets">CSS</abbr>-based design. Select any style sheet from the list to load it into this page.</p>
```
Comment (ghost) [inline pristine-zen-garden.html]: You should indent text/content from code tags :)

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        <div class="design-archives" id="design-archives">
+
+          <h3 class="archives">Archives:</h3>
+
+          <nav role="navigation">
+
+            <ul>
+              <li class="next">
+                <a href="/214/page3">
+                Next Designs <span class="indicator">&rsaquo;</span>
```
Comment (ghost) [inline pristine-zen-garden.html]: Inline elements such as span shouldn't be indented. They should be treated as text/content.

File: pristine-zen-garden.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+      <footer>
+        <a href="http://validator.w3.org/check/referer" title="Check the validity of this site&#8217;s HTML" class="zen-validate-html">HTML</a>
+        <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
+        <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike." class="zen-license">CC</a>
+        <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
+        <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
+      </footer>
+
+    </div>
```
Comment (steph-j) [inline pristine-zen-garden.html:108]: This div is closing the "page-wrapper" but it isn't on the same indent

File: pristine-zen-garden.html (hunk 1 above)
Comment (steph-j) [inline pristine-zen-garden.html]: Your p's need another enter to make them standout as per the resource. 
p
content
/p

File: pristine-zen-garden.html (hunk 3 above)
Comment (josh-jacobsen) [inline pristine-zen-garden.html:108]: That div closes "main supporting" from line 39

File: pristine-zen-garden.html (hunk 2 above)
Comment (josh-jacobsen) [inline pristine-zen-garden.html]: Thanks; will edit

File: pristine-zen-garden.html (hunk 3 above)
Comment (steph-j) [inline pristine-zen-garden.html:108]: Oh ah, ur right. lol. Don't mind me.

**Pull request merged**